etl/steps/data/garden/energy_institute/2023-06-26/statistical_review_of_world_energy.meta.yml

```diff
@@ -0,0 +1,709 @@
+tables:
+  statistical_review_of_world_energy:
```

Comment:
I'm curious about how you produce this metadata? Is it copied over from the BP statistical energy metadata?

Reply:
I took most of it from what we had in importers, I don't know if it was Bobbie who prepared it. Most of the texts seem to come from footnotes in the sheets of the main excel data file (hard to parse).